server/gc/src/main/java/org/apache/accumulo/gc/GCRun.java

```diff
@@ -87,7 +87,9 @@
  * A single garbage collection performed on a table (Root, MD) or all User tables.
  */
 public class GCRun implements GarbageCollectionEnvironment {
+  // loggers are not static to support unique naming by level
   private final Logger log;
+  private static String fileActionPrefix = "FILE-ACTION:";
```

Comment:
I'm not sure why we need this variable. You could just add "FILE-ACTION" at the beginning of each log message. The only reason to make it a variable is in the case that you think it would change in a future version.

Reply:
My opinion: having it makes it explicit and also easier to track down with a show usages if ever need.  And if it was me, making sure its `spelt` the same everywhere ;-)

Reply:
@EdColeman you and I had the exact same thought on this. 
I had created it as a variable to ensure that misspelling did not occur between log messages since this would be utilized by a regex.

Reply:
Since it's used in so many places, it probably makes sense to keep it as a variable. But, it should be final:

```suggestion
  private static final String fileActionPrefix = "FILE-ACTION:";
```

Also, I'm not sure if "FILE-ACTION:" is a great prefix, or if we need it at all... isn't the Logger name distinct enough to filter on?

Reply:
There is a desire to filter GC logs further and only select log messages related to the deletion of a file. 
This prefix standardizes the log messages related to file deletions and allows them to be easily selected.

I used `FILE-ACTION` as an attempt to make it generic as opposed to sometime tightly linked to `DELETE`. 
If you have a different suggestion I'm open to it.